src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -44,16 +43,6 @@ class SideNav extends Component {
 
     // Escape early if the item has children.
     if (hasChildren) {
-      // Flip the item's expanded property.
-      this.setState({
-        navItemsLookup: {
-          ...navItemsLookup,
-          [id]: {
-            ...navItemsLookup[id],
-            expanded: !expanded,
-          },
-        },
-      });
```

Comment:
What do you think about keeping this functionality in case we decide to bring it back later?

We could add a prop that enables the expand/collapse behavior instead.